go/extra/stats/.gitignore

```diff
@@ -0,0 +1 @@
+extra/stats
```

Comment:
Change parent makefile to use `basename` on the finale binary filename instead of creating additional subdirectories?